flocker/volume/test/filesystemtests.py

```diff
@@ -618,7 +620,11 @@ def got_volumes(copied):
                     with from_filesystem.reader() as reader:
                         with to_filesystem.writer() as writer:
                             data = reader.read()
-                            writer.write(data[1:])
+                            try:
+                                writer.write(data[1:])
+                            except IOError as e:
+                                if e.errno != errno.EPIPE:
```

Comment:
Please add a comment explaining why this is not a real error. 

Suggestion:

```python
# Sometimes the reader will quit before the writer is done writing. This is OK because ____.
```